addons/mail/models/mail_message_translation.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MessageTranslation(models.Model):
+    _name = 'mail.message.translation'
+    _description = 'Message Translation'
+
+    message_id = fields.Many2one(
+        'mail.message', 'Message', required=True, index=True, ondelete='cascade',
+    )
+    lang = fields.Char(
+        'Target Language',
+        help='Shortened language code used as target for the translation request.',
+    )
```

Comment:
Might be worth having a unique index for these two rather than just index on message

Reply:
Added an index on the second field used for searching.